Server/controllers/articles.controller.js

```diff
@@ -37,6 +37,49 @@ class Articles {
       })
       .catch((err) => Response.error(res, 500, "Internal Server Error"));
   };
+
+  static update = async (req, res) => {
+    Article.findOne({ _id: req.params.id })
+      .then((article) => {
+        if (article) {
+          if (req.body.title) {
+            article.title = req.body.title;
+          }
+          if (req.body.content) {
+            article.body = req.body.content;
+          }
+          if (req.body.imageUrl) {
+            article.imageUrl = req.body.imageUrl;
+          }
```

Comment:
@elvisiraguha You can separate variables checking & validation from the actual operation processing by validating, checking if variables exist in the middleware (you can use joi package), then in your controller, you'll just have to make business operations.